checkov/cdk/checks/typescript/RedShiftSSL.yaml

```diff
@@ -0,0 +1,22 @@
+metadata:
+  version: 0.2
+  approach: define failing
+  id: CKV_AWS_369
+  name: Ensure Redshift uses SSL
+  category: ENCRYPTION
+  framework: cdk
+scope:
+  languages:
+    - javascript
+    - typescript
+definition:
+  or:
+  - pattern: new $REDSHIFT.CfnClusterParameterGroup(<ANY>)
+    conditions:
+      - not_pattern: |
+          "new $REDSHIFT.CfnClusterParameterGroup(<ANY>, require_ssl: 'true', <ANY>)"
```

Comment:
is the ' ' around the true necessary ?

Reply:
https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_redshiftserverless.CfnWorkgroup.ConfigParameterProperty.html
parameter type is str